### ISSUE TITLE
Are some Voltage Source nodes inverted in the examples?

### ISSUE THREAD

**Issue by GastonMazzei** [FabriceSalvaire/PySpice#243]
Hello, Im running PySpice installed globally while using Python 3.7 and ubuntu 18.04. The issue I want to adress is the fact that running the examples produced circuits with voltage sources with the nodes inverted. E.g. Example 8.13.2 "Capacitive Half Wave Rectification Post Zener",  _(TLDR at the end)_
A similar code that illustrates the idea:
```

from PySpice.Spice.Netlist import Circuit
circuit_object = Circuit("circuit_name") 
circuit_object.V('source', circuit_object.gnd, 'input', 'DC 5V')
print(circuit_object)

```
EXPECTED: 
_.title circuit_name
Vsource **input 0** DC 5V_

ACTUAL BEHAVIOUR:
_title circuit_name
Vsource **0 input** DC 5V_

Actual code's line from that example was:
`ac_line = circuit.AcLine('input', circuit.gnd, 'L', rms_voltage=230@u_V, frequency=50@u_Hz)
`

**And the full code from example 8.13.2 is:** 
```
import matplotlib.pyplot as plt

import PySpice.Logging.Logging as Logging
logger = Logging.setup_logging()

from PySpice.Doc.ExampleTools import find_libraries
from PySpice.Probe.Plot import plot
from PySpice.Spice.Library import SpiceLibrary
from PySpice.Spice.Netlist import Circuit
from PySpice.Unit import *

libraries_path = find_libraries()
spice_library = SpiceLibrary(libraries_path)

circuit = Circuit('Capacitive Half-Wave Rectification (Post Zener)')

circuit.include(spice_library['1N4148'])
# 1N5919B: 5.6 V, 3.0 W Zener Diode Voltage Regulator
circuit.include(spice_library['d1n5919brl'])

ac_line = circuit.AcLine('input', circuit.gnd, 'L', rms_voltage=230@u_V, frequency=50@u_Hz)
circuit.R('in', 'L', 1, 470@u_Ω)
circuit.C('in', 1, 2, 470@u_nF)
circuit.X('Dz', 'd1n5919brl', circuit.gnd, 2)
circuit.X('D', '1N4148', 2, 'out')
circuit.C('', circuit.gnd, 'out', 220@u_uF)
circuit.R('load', circuit.gnd, 'out', 1@u_kΩ)

# print circuit.nodes

# Simulator(circuit, ...).transient(...)
simulator = circuit.simulator(temperature=25, nominal_temperature=25)
analysis = simulator.transient(step_time=ac_line.period/200, end_time=ac_line.period*10)

figure, ax = plt.subplots(figsize=(20, 10))

ax.plot(analysis['L'] / 100)
ax.plot(analysis.out)
###ax.plot((analysis.out - analysis['L']) / 100)
###ax.plot(analysis.out - analysis['2'])
###ax.plot((analysis['2'] - analysis['1']) / 100)
# or:
#   plt.ax.plot(analysis.out.abscissa, analysis.out)
ax.legend(('Vin [V]', 'Vout [V]'), loc=(.8,.8))
ax.grid()
ax.set_xlabel('t [s]')
ax.set_ylabel('[V]')

plt.tight_layout()
plt.show()
```

MORE INFO: 
We are getting what we requested; the ordering is as expected. It's not a bug in the library but in the examples. 
Let's see the FORMAT SPEC for SPICE files according to Berkeley: 
![github_issue_A](https://user-images.githubusercontent.com/50888364/87842520-66d92800-c883-11ea-9ddb-fc37f965c8b2.png)

Nevertheless I must mention that, for example, Diodes in the examples DO COMFORM to the (+) (-) specs. of the SPICE algorithm; I hope you don't mind I've marked it on top of your tutorial: 
![github_issue_B](https://user-images.githubusercontent.com/50888364/87842563-ab64c380-c883-11ea-98d9-8bdbd7f12cd3.png)

**TLDR: I firmly believe that Voltage sources in the examples have their terminals inverted. In case I've missed something I may apologize in advance and in any case thank you for your time.**

**Comment by FabriceSalvaire** [comment]
You are right, it is a bit confusing. But simulation is right since it is an AC source.
